src/test/java/fi/hsl/transitdata/omm/models/AlertStateTest.java

```diff
@@ -209,9 +209,8 @@ private AlertState createModifiedAlertState(final List<Bulletin> secondBulletins
     public void testLastModified() throws Exception {
         List<Bulletin> bulletins = readDefaultTestBulletins();
         AlertState state = new AlertState(bulletins);
-        Optional<LocalDateTime> maybeDt = state.lastModified();
-        assertTrue(maybeDt.isPresent());
-        String dt = DAOImplBase.OMM_DT_FORMATTER.format(maybeDt.get());
+        LocalDateTime lastModified = state.lastModified();
+        String dt = DAOImplBase.OMM_DT_FORMATTER.format(lastModified);
         assertEquals("2019-05-08 09:44:54", dt);
```

Comment:
This and another test is failing:
https://travis-ci.org/HSLdevcom/transitdata-omm-alert-source